docs/api/shell.md

```diff
@@ -20,13 +20,13 @@ The `shell` module has the following methods:
 
 * `fullPath` String
 
-Show the given file in a file manager. If possible, select the file.
+Show the given file in a file manager. If possible, select the file. Expects `fullPath` to be a fully-specified path (`/users/<username>/path/to/file` or `~/path/to/file`). Relative paths (`./file`) do not work.
```

Comment:
Node.js can retrieve the home path on all 3 platforms using:
```javascript
const os = require('os');
const homepath = os.homedir();
```
If you are looking to ensure the home path is used and fully specified this might be the safer way of doing it.